.golangci.yml

```diff
@@ -74,6 +74,10 @@ linters-settings:
         disabled: true
       - name: cognitive-complexity
         disabled: true
+      - name: comment-spacings
+        disabled: true
+      - name: use-any
```

Comment:
https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#use-any

Reply:
sounds like a good rule, why can't we enable this?

Reply:
enabled this in latest changes 👍